Updated file classifier test

antran-17 opened the pull request:
## 📝 Description

* Updated tests for file classifier to reflect changes made to the input of isText and isCode functions

**Closes:** # (issue number)

---

## 🔧 Type of Change

- [ ] 🐛 Bug fix (non-breaking change that fixes an issue)
- [ ] ✨ New feature (non-breaking change that adds functionality)
- [ ] 💥 Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] 📚 Documentation added/updated
- [x] ✅ Test added/updated
- [ ] ♻️ Refactoring
- [ ] ⚡ Performance improvement

---

## 🧪 Testing

Tested on Docker container with the updated file_classifier code, and all tests passed locally.

In Docker container terminal, run `pytest tests_backend/file_classifier_test.py`, and confirm to run all file classifier tests.

---

## ✓ Checklist

- [ ] 🤖 GenAI was used in generating the code and I have performed a self-review of my own code
- [ ] 💬 I have commented my code where needed
- [ ] 📖 I have made corresponding changes to the documentation
- [x] ⚠️ My changes generate no new warnings
- [x] ✅ I have added tests that prove my fix is effective or that my feature works and tests are passing locally
- [ ] 🔗 Any dependent changes have been merged and published in downstream modules
- [ ] 📱 Any UI changes have been checked to work on desktop, tablet, and/or mobile

---

## 📸 Screenshots

> If applicable, add screenshots to help explain your changes

<details>
<summary>Click to expand screenshots</summary>

<!-- Add your screenshots here -->

</details>

maddydeg reviewed (approved):
Apologies, just realized those expected updates are in PR https://github.com/COSC-499-W2025/capstone-project-team-12/pull/105 and are currently reflected in the error-handling branch! These tests pass when using the updated file classifier code within error-handling and are good to merge!

This was originally left as a comment under my initial response, I have removed the other to ensure that this was an approval.

antran-17 commented:
> Apologies, just realized those expected updates are in PR #105 and are currently reflected in the error-handling branch! These tests pass when using the updated file classifier code within error-handling and are good to merge!
> 
> This was originally left as a comment under my initial response, I have removed the other to ensure that this was an approval.

Sorry, I updated the testing reproduction description to just test the file classifier now!
